sencha-workspace/SlateDemonstrationsStudent/app/view/Dashboard.js

```diff
@@ -5,8 +5,9 @@ Ext.define('SlateDemonstrationsStudent.view.Dashboard', {
         'Slate.cbl.Util',
 
         'Slate.cbl.widget.Popover',
-        'Slate.cbl.view.student.CompetencyCard',
-        'Slate.cbl.view.student.RecentProgress',
+        
```

Comment:
@aliyeysides you're committing trailing whitespace here, be sure to configure your editor that trims trailing whitespace on save